doc/modules/reference.rst

```diff
@@ -277,3 +277,31 @@ uses.
    clean
 
 
+.. _connectivity_ref:
+
+:mod:`nilearn.connectivity`: Functional Connectivity
+====================================================
+
+.. automodule:: nilearn.connectivity
+   :no-members:
+   :no-inherited-members:
+
+**Classes**:
+
+.. currentmodule:: nilearn.connectivity.connectivity_matrices
```

Comment:
I think that we should use as a currentmodule nilearn.connectivity, and the 'nilearn.connectivity.connectivity_matrices' module should pretty much be hidden from users.